src/api/routes/channels/#channel_id/webhooks.ts

```diff
@@ -58,7 +58,7 @@ router.post(
 		if (webhook_count > maxWebhooks)
```

Comment:
all server side numeric limits should be handle to handle the unlimited case:
```suggestion
		if (maxWebhooks && (webhook_count > maxWebhooks))
```